### DISCUSSION TITLE
[Adonis V5]: API Testing in Adonis v5

### DISCUSSION THREAD

**Discussion by aayush123** [adonisjs/core#1704] [Help] [answered]
Greetings,

I'm working on a new REST Backend application and have decided to go forth with Adonis v5.
As far as I believe, the testing packages have not yet been ported over to v5.
So I was wondering if anyone has used something else to test the APIs in v5 and can point me in some direction on what to you. while I wait for the package to be ported?

Thanks!

**Comment by fnoquiq** [comment, marked as answer]
Check it: https://preview.adonisjs.com/blog/running-tests-in-adonisjs-v5
